mobile/src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,20 @@
+import { StatusBar } from 'expo-status-bar';
+import { Button, View } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+
+export default function App() {
+  const value: string = "hello, world";
```

Comment:
Unused variables should be removed.

Reply:
wait a second, these are changes from main but I will comply